### DISCUSSION TITLE
May I ask how to add link tracking code for the OpenTelemetry specification in the Go project？

### DISCUSSION THREAD

**Discussion by jettjia** [openobserve/openobserve#611] [Q&A]
请问在go项目中，如何增加OpenTelemetry规范的链路追踪代码？目前只看到有日志部分的。

(Translated - In the go project, how to add the link tracking code of the OpenTelemetry specification? Currently only see the log part.)

May I ask how to add link tracking code for the OpenTelemetry specification in the Go project？

**Comment by hengfeiyang** [comment]
Good topic, we are working on traces, I will ping you when the feature can work.

**Comment by prabhatsharma** [comment]
What do you mean by link tracking code ? Can you provide the Opentelemetry specification link?
